src/SprykerEco/Zed/Computop/Business/ComputopFacade.php

```diff
@@ -183,6 +183,78 @@ public function saveIdealInitResponse(QuoteTransfer $quoteTransfer)
         return $quoteTransfer;
     }
 
+    /**
+     * {@inheritdoc}
+     *
+     * @api
+     *
+     * @param \Generated\Shared\Transfer\QuoteTransfer $quoteTransfer
+     *
+     * @return \Generated\Shared\Transfer\QuoteTransfer
+     */
+    public function saveCreditCardInitResponse(QuoteTransfer $quoteTransfer)
+    {
+        $this->getFactory()->createCreditCardResponseSaver()->handle(
+            $quoteTransfer
+        );
+
+        return $quoteTransfer;
```

Comment:
Please return `quoteTransfer` from `->handle` call.
This way of Facade method looks suspicious due to `change by reference` behavior.

Reply:
Please convert it to 'return $this->getFactory()->createCreditCardResponseSaver()->handle(
+            $quoteTransfer
+        );`

Reply:
And check other places as well